setup.py

```diff
@@ -23,10 +23,13 @@
 
 import codecs
 import os
+import os.path
```

Comment:
This isn't needed. `os` import do the same work.

Reply:
You're right, sorry.